### PR TITLE
Feat/readme file templates

### LINKED ISSUE #202: Readme File Templates
## Project Readme Files Template

Let's set a default readme file template for new repositories as new document under `developer guidelines`.

Here is a good template we can use to base projects off:
https://github.com/othneildrew/Best-README-Template

Here is an example used in EOS CR repositories 

https://github.com/eoscostarica/evodex/blob/develop/README.md

Also check out some cool examples:

https://github.com/matiassingers/awesome-readme

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #202  

### Steps to test
1. Open developer guidelines tag
1. Check README template tag exist
1. Check content is right

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/readme-file-template.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+Currently evodex is still in beta pre-release *v0.1.0-beta.xx*
+
+We release the production software version using github tags following Semver.
+
+We also use the git commit messages to generate the project change log.
+
+Using the yarn version command you can update the version of your package via the command line.
+:::
+
+## Teach Stack
```
Comment (xavier506) [inline docs/readme-file-template.md:65]: @leisterfrancisco   'Tech Stack'

File: docs/readme-file-template.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+Please Read EOS Costa Rica's [Open Source Contributing Guidelines](https://guide.eoscostarica.io/docs/open-source-guidelines/) for more information about programming conventions.
+
+If you find a bug, please report big and small bugs by **opening an issue** `<link to issue project section>`
+
+## Contributors
+
+List the people who work in the project.
+
+## About Company
+
```
Comment (xavier506) [inline docs/readme-file-template.md:179]: Let's add the logo

File: docs/readme-file-template.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+Please Read EOS Costa Rica's [Open Source Contributing Guidelines](https://guide.eoscostarica.io/docs/open-source-guidelines/) for more information about programming conventions.
+
+If you find a bug, please report big and small bugs by **opening an issue** `<link to issue project section>`
+
+## Contributors
+
+List the people who work in the project.
+
+## About Company
```
Comment (xavier506) [inline docs/readme-file-template.md:178]: About EOS Costa Rica

**Review by xavier506 — CHANGES_REQUESTED** [review]
Please see my comments

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by steph222 — APPROVED** [review]

**Review by edgarEOS — APPROVED** [review]
👍

**Pull request merged**